fh-ios-sdk.xcodeproj/project.pbxproj

```diff
@@ -496,7 +486,7 @@
 				B3D7C48E15D28BE30019262E /* Sources */,
 				B3D7C48F15D28BE30019262E /* Frameworks */,
 				B3D7C49015D28BE30019262E /* Resources */,
-				B3D7C49115D28BE30019262E /* ShellScript */,
```

Comment:
..maybe xCode updating automatically for 6.4, not 100% sure where this came from